### ISSUE TITLE
[Critical] Information or relationship only presented visually

### ISSUE THREAD

**Issue by ntallman** [APTrust/registry#112]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87922 |
| **Checkpoint** | 1.3 Adaptable › Semantics (1.3.1.a) |
| **Impact** | Critical |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 06 Alerts |
| **Test Case Name** | AP Trust Registry (Demo) |
| **URL** | <https://demo.aptrust.org/alerts> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Product** | Website |
| **Date Reported** | 05/17/26 at 2:05 pm |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | kim.casey@ablr360.com |

## Description

Information, structure, or relationship is presented visually but is not conveyed programmatically or in text.
If the User has unread Alerts, there is a red dot indicating action needed, but this is not conveyed via a screen reader.

## Recommendation

Rule : 
Information, structure, or relationship that is presented visually MUST be conveyed programmatically or in text.

How To Fix: 
Fix this issue by doing ONE OR MORE of the following:
1. Use a text-based method to convey the meaning of the visual information or relationship. The text can be provided for screen readers only or be visible on screen.
2. When available in HTML, use appropriate semantic HTML elements.
3. Use ARIA to provide programmatic information for screen readers. (Caution: Be certain you know which ARIA attributes can be used on which HTML elements and test the results with screen readers.)

Reference: 
Invisible Content: https://webaim.org/techniques/css/invisiblecontent/

Background: 
People who are blind cannot rely on the visual appearance and layout of a page to understand information, relationships, or structure that is conveyed only visually. In order to convey visual information to screen reader users it must be also provided programmatically or in text. This will allow screen reader users to understand information, relationships, or structure that is apparent visually.

## Source Code

```
<a href="/alerts" class="alert-icon has-notifications">
    <span class="material-icons md-30" aria-hidden="true">notifications</span>
    <span class="is-sr-only">Alerts</span>
  </a>
```

## Screenshots

![06_unread dot](https://raw.githubusercontent.com/APTrust/registry/master/docs/accessibility-audit/screenshots/87922/06_unread_dot.png)
